libs/pika/functional/tests/unit/function_object_size.cpp

```diff
@@ -1,238 +0,0 @@
-////////////////////////////////////////////////////////////////////////////////
```

Comment:
This test for example tests non-serialization functionality as well and should stay (but the serialization functions can be removed of course).

Reply:
good it was probably the one we talked about, I will restore the non-serialization functionalities :ok_hand: